mtaos_sim/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Dockerfile to execute a base machine with the MTAOS simulation environment
 
-FROM lsstts/aos_aoclc:w_2020_06_sal
+FROM lsstts/aos_aoclc:w_2020_15_sal
```

Comment:
Instead of having to update this tag even time could you use a build argument instead?

```
ARG image_tag

FROM lsstts/aos_aoclc:${image_tag}
```

Reply:
Done in the commit: 75bc978. Thanks!